api/src/main/java/edu/cornell/mannlib/vitro/webapp/auth/checks/SparqlSelectQueryResultsChecker.java

```diff
@@ -26,15 +27,19 @@
 import org.apache.jena.rdf.model.RDFNode;
 
 public class SparqlSelectQueryResultsChecker {
+    private static final String VAR_PROFILE_URI = "profileUri";
+    private static final String VAR_EXTERNAL_AUTH_ID = "externalAuthId";
+    private static final String VAR_MATCHING_PROPERTY_URI = "matchingPropertyUri";
+    private static final String VAR_OBJECT_URI = "objectUri";
     private static final Log log = LogFactory.getLog(SparqlSelectQueryResultsChecker.class);
 
     public static boolean sparqlSelectQueryResultsContain(Check check, AuthorizationRequest ar, String[] inputValues) {
-        String queryTemplate = check.getConfiguration();
-        if (StringUtils.isBlank(queryTemplate)) {
-            queryTemplate = check.getValues().getSingleValue();
+        String query = check.getConfiguration();
+        if (StringUtils.isBlank(query)) {
+            query = check.getValues().getSingleValue();
         }
-        if (StringUtils.isBlank(queryTemplate)) {
-            log.error("SparqlQueryContains template is empty");
+        if (StringUtils.isBlank(query)) {
+            log.error("Sparql query is empty.");
```

Comment:
Although this is just renaming variable, it looks too me as strange code, and I think we might fix this in this PR. I think the second if should be inside the first if clause. In the case `check.getConfiguration()` returns not a blank there is no reason to two times check clause `StringUtils.isBlank(query)`

Reply:
fixed